.github/scripts/integrate-rc-to-develop.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -euo pipefail
+
+readonly TODAY=$(date +%m%d)
+readonly DATE_DISPLAY=$(date +%Y-%m-%d)
+
+branch_exists() {
+    local branch=$1
+    git show-ref --verify --quiet "refs/remotes/origin/$branch"
+}
+
+create_merge_branch() {
+    local RC_BRANCH=$1
+    local MERGE_BRANCH=$2
+    
+    echo "Creating new merge branch: $MERGE_BRANCH from $RC_BRANCH"
+    git fetch origin
+    git checkout "origin/$RC_BRANCH"
+    git checkout -b "$MERGE_BRANCH"
+
+    echo "Merging develop into $MERGE_BRANCH to detect conflicts"
+    if git merge origin/develop --no-commit --no-ff; then
+        echo "Clean merge - no conflicts detected"
+        git commit -m "Merge develop into $MERGE_BRANCH"
+    else 
+        echo "Merge conflicts detected"
+        if git status --porcelain | grep -q "package.json"; then 
+            echo "Conflicts in package.json detected, auto-resolving"
+            git checkout --theirs package.json
+            git add package.json
+        fi 
+
+        if git status --porcelain | grep -q "^UU\|^AA\|^DD"; then
+            git commit -m "Merge develop into $MERGE_BRANCH (manual resolution needed)"
```

Comment:
Do we want to commit with the merge conflicts? Will they still be visible in the PR if so?